WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -170,6 +166,17 @@ - (void)viewWillDisappear:(BOOL)animated
     self.resultsController.delegate = nil;
 }
 
+- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation
```

Comment:
This works but you'll notice a bit of a jump after rotation as the update height is applied.  Using `- (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration` makes for a smoother transition.